docs/configuration.md

```diff
@@ -113,12 +96,12 @@ The mongodb post used by udata.
 
 **default**: `udata`
 
-The mongodb database used by udata. When testing, the test database will use the same name suffixed by `-test`
+The mongodb database used by udata.
+During tests, the test database will use the same name suffixed by `-test`
 
-See [the official Flask-MongoEngine documentation](https://flask-mongoengine.readthedocs.org/en/latest/)
+See [the official Flask-MongoEngine documentation][flask-mongoengine-doc]
```

Comment:
Is this an internal link to _readthedocs_? In which case it creates a hard dependency on the doc beeing read on the readthedocs website. Should we not just keep the markdown doc readable from anywhere?